clients/database/migrations/main.go

```diff
@@ -0,0 +1,15 @@
+// when this package is loaded the package directory
+// will be searched for any go or sql migration files and
+// register for them to the proxy service to run
+// https://bun.uptrace.dev/guide/migrations.html#go-based-migrations
+package migrations
+
+import "github.com/uptrace/bun/migrate"
+
+var Migrations = migrate.NewMigrations()
```

Comment:
I believe the embed for the sql migrations are missing? Or I am not seeing how they are being embedded in the binary

Reply:
they are copied into the container along with the golang source code and the library is able to find them by walking the filesystem

Reply:
Gotchya, but that won't work for production docker containers where the build container is separate from the run container?

Reply:
imo overly pared down production containers are security theatre the majority of the time. If someone is able to get a shell/run commands in a companies production container, that company probably has bigger fish to fry then the attacker having access to public open source code on the container

That being said I'll add a ticket for figuring out how to embed these

Reply:
embeded the sql files